### PR TITLE
XWIKI-10731: Make Create Wiki responsive

### PR DESCRIPTION
**Pull request by gabrielasmeria**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml [hunk 1]
```diff
@@ -191,113 +191,115 @@
...
-              &lt;/dd&gt;
-            &lt;/dl&gt;
-          &lt;/div&gt;
-          ### WIKI ALIAS ###
-          #if(!$services.wiki.isPathMode())
+        &lt;div class="row"&gt;
+          ### WIZARD FIRST HALF ###
+          &lt;div class = "col-xs-12 col-sm-6 col-md-6"&gt;
+            ### PRETTY NAME ###
             &lt;div&gt;
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml]: is this div needed?

File: xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml [hunk 2]
```diff
@@ -191,113 +191,115 @@
...
-              &lt;dt&gt;
-                &lt;label for="description"&gt;$services.localization.render('platform.wiki.prop.description')&lt;/label&gt;
-                &lt;span class="xHint"&gt;$services.localization.render('platform.wiki.create.wizard.desc.description')&lt;/span&gt;
-              &lt;/dt&gt;
-              &lt;dd&gt;
-                &lt;textarea id="description" name="description" rows="7" cols="45"&gt;$!escapetool.xml($request.getParameter('description'))&lt;/textarea&gt;
-              &lt;/dd&gt;
-            &lt;/dl&gt;
+            ### WIKI NAME ###
+            &lt;div&gt;
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml]: needed?

File: xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml [hunk 3]
```diff
@@ -191,113 +191,115 @@
...
                 &lt;dt&gt;
-                  &lt;label for="wikialias"&gt;$services.localization.render('platform.wiki.prop.wikialias')#required()&lt;/label&gt;
-                  &lt;span class='xHint'&gt;$services.localization.render('platform.wiki.create.wizard.desc.wikialias')&lt;/span&gt;
+                  &lt;label for="prettyname"&gt;$services.localization.render('platform.wiki.prop.wikiprettyname')#required()&lt;/label&gt;
+                  &lt;span class="xHint"&gt;$services.localization.render('platform.wiki.create.wizard.desc.wikiprettyname')&lt;/span&gt;
+                  &lt;span id="wikiprettynamevalidation" class="xErrorMsg"&gt;&lt;/span&gt;
                 &lt;/dt&gt;
                 &lt;dd&gt;
-                  &lt;input size="40" id="wikialias" name="wikialias" type="text"  value="$!escapetool.xml($request.wikialias)"/&gt;
+                  &lt;input size="60" id="prettyname" name="wikiprettyname" type="text" value="$!escapetool.xml($request.getParameter('wikiprettyname'))"/&gt;
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml]: We could use the same size for input type text. In http://platform.xwiki.org/xwiki/bin/view/DevGuide/VerticalForms, the size is recommended to '60'

File: xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml [hunk 4]
```diff
@@ -191,113 +191,115 @@
...
+                &lt;/dt&gt;
+                &lt;dd&gt;
+                  &lt;input size="40" id="wikiname" name="wikiname" type="text"  value="$!escapetool.xml($request.wikiname)"/&gt;
+                  &lt;span id="wikinamevalidation" class="xErrorMsg"&gt;&lt;/span&gt;
+                &lt;/dd&gt;
+              &lt;/dl&gt;
+            &lt;/div&gt;
+            ### WIKI ALIAS ###
+            #if(!$services.wiki.isPathMode())
+              &lt;div&gt;
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml]: needed?

File: xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml [hunk 5]
```diff
@@ -191,113 +191,115 @@
...
+                    &lt;span class='xHint'&gt;$services.localization.render('platform.wiki.create.wizard.desc.wikialias')&lt;/span&gt;
+                  &lt;/dt&gt;
+                  &lt;dd&gt;
+                    &lt;input size="40" id="wikialias" name="wikialias" type="text"  value="$!escapetool.xml($request.wikialias)"/&gt;
+                  &lt;/dd&gt;
+                &lt;/dl&gt;
+              &lt;/div&gt;
+            #end
+            ### DESCRIPTION ###
+            &lt;div&gt;
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml]: again?

File: xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml [hunk 6]
```diff
@@ -328,111 +330,113 @@
...
-        &lt;div class="half"&gt;
-          ### OWNER ###
-          &lt;div&gt;
-            ## Make this section available only to admins, so that only admins can create wikis in the name of other users.
-            #if ($hasAdmin)
+        &lt;div class="row"&gt;
+          ### WIZARD FIRST HALF ###
+          &lt;div class="col-xs-12 col-sm-6 col-md-6"&gt;
+            ### OWNER ###
+            &lt;div&gt;
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml]: ?

File: xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml [hunk 7]
```diff
@@ -191,113 +191,115 @@
         &lt;input type="hidden" name="membershipType" value="$!escapetool.xml($request.getParameter('membershipType'))" /&gt;
         &lt;input type="hidden" name="userScope" value="$!escapetool.xml($request.userScope)" /&gt;
         #addMembersAsHiddenInput()
-        ### WIZARD FIRST HALF ###
-        &lt;div class="half"&gt;
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml]: Just curious, how does this look on Colibri?

**Comment by evalica** [comment]
So after thinking more about this issue, and thanks @Enygma2002 for reminding me, `.half` is extended in Flamingo in `grid.less`, see https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/grid.less#L12

So in theory, the layout should already be responsive, based on the backwards compatibility rule. 
So, actually the solution for this issue is: 
- do not change the .half classes, since they should work;
- but see what causes the different `width`. And the problem is that this was manually set in `skin/resources/uicomponents/wizard/wizard.css` to 48%. So we should remove those manually overrides of the skin defaults. 

The comments about extra `div`s and `size`s are still relevant and we should profit the occasion to fix them. WDTY?

**Comment by evalica** [comment]
Btw, `wizard.css` should be looked at to see if there are other deprecated rules.

**Comment by gabrielasmeria** [comment]
I didn't add those divs or change those sizes, I just moved the code from one place to another, that's why it looks like I added them. 
I tested to see if is still responsive when removing width: 48% and it is. So, I will close this PR and make a much cleaner one.
Thanks!

**Pull request closed without merging**